lib/crontab/cron_expression/parser.ex

```diff
@@ -72,32 +72,31 @@ defmodule Crontab.CronExpression.Parser do
       iex> Crontab.CronExpression.Parser.parse "* * * * *"
       {:ok,
         %Crontab.CronExpression{day: [:*], hour: [:*], minute: [:*],
-        month: [:*], weekday: [:*], year: [:*]}}
+        month: [:*], weekday: [:*], year: [:*], on_ambiguity: [:later]}}
 
       iex> Crontab.CronExpression.Parser.parse "* * * * *", true
       {:ok,
         %Crontab.CronExpression{extended: true, day: [:*], hour: [:*], minute: [:*],
-        month: [:*], weekday: [:*], year: [:*], second: [:*]}}
+        month: [:*], weekday: [:*], year: [:*], second: [:*], on_ambiguity: [:later]}}
 
       iex> Crontab.CronExpression.Parser.parse "fooo"
       {:error, "Can't parse fooo as minute."}
 
   """
-  @spec parse(binary, boolean) :: result
-  def parse(cron_expression, extended \\ false)
+  @spec parse(binary, boolean, [CronExpression.ambiguity_opt()]) :: result
+  def parse(cron_expression, extended \\ false, ambiguity_opts \\ [:later])
```

Comment:
It could be helpful to change the second attribute to be an option keyword list if we ever introduce more options.

Something like this:

```elixir
@type options() :: [extended: boolean(), on_ambiguity: [CronExpression.ambiguity_opt()]]

@spec parse(cron_expression :: binary(), options :: options()) :: result
def parse(cron_expression, options \\ [])
def parse(cron_expression, extended) when is_boolean(extended) do
  IO.warn("#{__MODULE__}.parse(#{inspect(cron_expression)}, #{inspect(extended)}) is deprecated, use #{__MODULE__}.parse(#{inspect(cron_expression)}, extended: #{inspect(extended)}) instead.")
  parse(cron_expression, extended: extended)
end
def parse("@" <> identifier, _options) do # ...
def parse (cron_expression, options) do
  options = Keyword.validate!(options, [extended: false, on_ambiguity: [:later])
  # options[:extended]
  # options[:on_ambiguity]
end

Reply:
This would entail a lot of changes in the code. Let's focus on getting the computations right first for this PR. I'd submit another PR to handle that chore.